### ISSUE TITLE
Create doctor detail pages (conventional routing)

### ISSUE THREAD

**Issue by JanLenoch** [Kentico/xperience-training-13#38]
### Who

- [x] Anonymous visitor
- [ ] Patient
- [ ] Doctor
- [ ] Content editor
- [ ] Marketer
- [ ] Administrator
- [ ] Developer

### Needs what

Pages with detailed information about each doctor.

### Why

Why is the feature needed?

### Issue expands upon

This issue expands upon another issue(s): #`issue number`

### Blockers

This issue is blocked by the following issue(s): #9
